code/modules/SCP/SCPs/SCP-527.dm

```diff
@@ -0,0 +1,39 @@
+/mob/living/carbon/human/scp_527
+	SCP = /datum/scp/scp_527
+	icon = 'icons/SCP/scp-527.dmi'
```

Comment:
fixed

Reply:
Not fixed

Reply:
@Tsurupeta we have a separate icon file for 106 as well - what's the standard?

Reply:
> @Tsurupeta we have a separate icon file for 106 as well - what's the standard?

and 049 too, its just a backup in case the body sprite thing doesn't work, or something IDK. I'm scared to remove it.

Reply:
> @Tsurupeta we have a separate icon file for 106 as well - what's the standard?

The standard is to not add useless code. 106 and 049 a giant piles of shitcode that nobody dares to touch, but new species should not look like them,